sql-plugin/src/main/301db/scala/com/nvidia/spark/rapids/shims/v2/Spark30XShims.scala

```diff
@@ -93,4 +93,6 @@ trait Spark30XShims extends SparkShims {
   }
 
   override def shouldFailDivOverflow(): Boolean = false
+
+  override def getTypeSigUtil(): TypeSigUtil = TypeSigUtilUntil320
```

Comment:
We don't need anything in the Shim layer to make this happen. You can just have a single class named TypeSigUtil. Then you have different versions in the appropriate directories.

Reply:
Yeah, You're correct. Thx. Done